src/components/ui/DataGrid/BulkActionDataGrid.svelte

```diff
@@ -214,6 +214,9 @@
           </ContextMenu.Item>
         </div>
       {/if}
+
+      <slot name="context-menu-alt" />
```

Comment:
If the comment about the context-menu results in combining the two context menu items into one group, then this slot can be removed.